cmd/libguestfs/entrypoint.sh

```diff
@@ -1,10 +1,11 @@
 #!/bin/bash -xe
 
-LIBGUESTFS_PATH=${LIBGUESTFS_PATH:=/tmp/guestfs}
-LIBGUEST_APPLIANCE=/usr/local/lib/guestfs/downloaded
-mkdir -p ${LIBGUESTFS_PATH}
-tar -Jxf ${LIBGUEST_APPLIANCE} -C ${LIBGUESTFS_PATH} --strip-components=1
+LIBGUESTFS_PATH=${LIBGUESTFS_PATH:=/usr/local/lib/guestfs/appliance}
```

Comment:
Very minor, but I think `:-` makes more sense than `:=` here.

See https://www.gnu.org/software/bash/manual/bash.html#Shell-Parameter-Expansion for the difference between the two.

Reply:
You are of course right! Done